packages/automerge-repo/src/index.ts

```diff
@@ -1,24 +1,12 @@
-export { DocCollection } from "./DocCollection.js"
-export {
-  DocHandle,
-  DocHandleChangeEvent,
-  DocHandlePatchEvent,
-  DocumentId,
-} from "./DocHandle.js"
-export {
-  ChannelId,
-  DecodedMessage,
-  NetworkAdapter,
-  NetworkAdapterEvents,
-  NetworkSubsystem,
-  Peer,
-  PeerId,
-} from "./network/NetworkSubsystem.js"
-export { Repo } from "./Repo.js"
-export {
-  StorageAdapter, //
-  StorageSubsystem,
-} from "./storage/StorageSubsystem.js"
-export {
-  CollectionSynchronizer as DependencyCollectionSynchronizer, //
-} from "./synchronizer/CollectionSynchronizer.js"
+export * from "./DocCollection"
+export * from "./DocHandle"
+export * from "./network/NetworkAdapter"
+export * from "./network/NetworkSubsystem"
+export * from "./Repo"
+export * from "./storage/StorageAdapter"
+export * from "./storage/StorageSubsystem"
+export * from "./synchronizer/CollectionSynchronizer"
+export * from "./types"
+
+// Q: is there a reason this is exported with a different name?
+export { CollectionSynchronizer as DependencyCollectionSynchronizer } from "./synchronizer/CollectionSynchronizer"
```

Comment:
we can just fix this -- this is a refactoring leftover.

also, let's keep the explicit exports -- it allows internal modules to export things for each other's use without leaking them into the public interface for the library

Reply:
✅